modules/internal/StringCasts.chpl

```diff
@@ -165,7 +165,7 @@ module StringCasts {
                                               numCodepoints=len);
   }
 
-  proc _cast(type t:string, x:chpl_anyreal) {
+  inline proc _cast(type t:string, x:chpl_anyreal) {
```

Comment:
I think it's a wash in this case, but any reason to not inline `_real_cast_helper` and leave the casts not-inlined (just thinking about a future case where we might copy a `_cast` routine again and inline without thinking about it too much?

Reply:
I agree that I don't think it matters that much. After seeing the potential issues with inline w.r.t `inline`. I think we should inline wrappers instead of what they wrap. I'd be a bit worried if that helper is used elsewhere and we'd end up with a lot of inlined callsites for that function.

That's my weak argument, anyway. I can do as you suggest, if you still that way.

Reply:
Yeah, I buy that argument. I'd also be ok with not inlining this cast as well and either having the double function call or hoping the backend inlines.